public/index.html

```diff
@@ -1,7 +1,8 @@
 <html>
   <head>
     <link rel="icon" href="data:;base64,iVBORw0KGgo=" />
-    <script src="BandwidthRtc.bundle.js"></script>
+    <!-- we recommend hosting this yourself, github is not always the best CDN -->
+    <script src="https://github.com/Bandwidth/webrtc-browser/releases/download/0.9.1/BandwidthRtc.bundle.js"></script>
     <script language="javascript">
       const basePath = "http://localhost:3000";
```

Comment:
I'm not sure it's possible due to the sandboxed environment of browsers but the port is static to `3000` here and dynamic to `LOCAL_PORT || 3000` on the server. Assuming it's not possible to set via an environment variable maybe a hardcoded value would be less confusing on the server as well.

Reply:
Good catch, I updated